src/utils/hashing.test.ts

```diff
@@ -26,7 +26,22 @@ const longText =
 
 describe('Murmur3', () => {
   it('makes x64 128 bit hash', () => {
-    expect(x64hash128('Hello, world')).toBe('ebd28b45027ab97477416103e3fff7b8')
+    const input = 'Hello, world, hi'
+    const inputLessThanChunk = 'Hello, world, h'
+    const inputGreaterThanChunk = 'Hello, world, hi!'
+    const inputGreaterThan2Chunks = 'Hello, world, hi, Hello, world, hi'
+
+    const nonAsciiInput = 'ňťŬŬůĬĠŷůŲŬŤĬĠŨũ' // 'Hello, world, hi' + 256 codepoints for each to make it non-ASCII
```

Comment:
What do you think if this string is computed from `input` instead of hard-coding?